### PR TITLE
[WebApp] InstallReminder header banner - added

### PR DESCRIPTION
**Pull request by vitto-moz**
https://www.notion.so/saladtech/Frontend-New-Chef-Download-Flow-4abee69d88a9417f89a7c3b6e56fe0bd?p=5e463409363c46788ff40b7f97815238&pm=s

image

### CODE REVIEW

File: packages/web-app/src/modules/home-views/NavigationBarContainer.tsx [hunk 1]
```diff
@@ -60,6 +63,7 @@ const mapStoreToProps = (store: RootStore): any => {
         isLoading={store.bonuses.pendingBonuses?.has(bonus.id)}
       />
     ) : undefined,
+    headerBannerContent: !wasWidgetLoggedInOnce && <InstallReminder justifyContent={'flex-end'} />,
```
Comment (Maks19) [inline packages/web-app/src/modules/home-views/NavigationBarContainer.tsx:66]:
```suggestion
    headerBannerContent: !wasWidgetLoggedInOnce && <InstallReminder justifyContent='flex-end' />,
```

File: packages/web-app/src/modules/home-views/components/InstallReminder.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { ArrowUp, Download } from '@saladtechnologies/garden-icons'
+import type CSS from 'csstype'
+import type { FunctionComponent } from 'react'
+import type { WithStyles } from 'react-jss'
+import withStyles from 'react-jss'
+
+const styles: () => Record<string, CSS.Properties> = () => ({
+  installReminderWrapper: {
+    width: '100%',
+    height: '55px',
```
Comment (Maks19) [inline packages/web-app/src/modules/home-views/components/InstallReminder.tsx:11]: Do we use pattern with string value for styling in WebApp?

File: packages/web-app/src/modules/home-views/components/InstallReminder.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  text: {
+    fontWeight: 500,
+    marginLeft: '8px',
+    marginRight: '8px',
+  },
+})
+
+interface InstallReminderProps extends WithStyles<typeof styles> {
+  justifyContent?: 'flex-start' | 'center' | 'flex-end'
```
Comment (Maks19) [inline packages/web-app/src/modules/home-views/components/InstallReminder.tsx:27]: `InstallReminder` is used only with 'flex-end' 
Do you still need this parameter?

File: packages/web-app/src/modules/home-views/components/InstallReminder.tsx (hunk 2 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/home-views/components/InstallReminder.tsx:11]: it's typing from `CSS.Properties`: 
image

`CSS.Properties` brings the ability of autocomplete, and I found it extremely useful

**Review by Maks19 — APPROVED** [review]

**Pull request merged**